sdk/python/feast/constants.py

```diff
@@ -17,4 +17,10 @@
 # Maximum interval(secs) to wait between retries for retry function
 MAX_WAIT_INTERVAL: str = "60"
 
-AWS_LAMBDA_FEATURE_SERVER_IMAGE = "feastdev/feature-server"
+AWS_LAMBDA_FEATURE_SERVER_IMAGE = "feastdev/feature-server:aws"
+
+# feature_store.yaml environment variable name for remote feature server
+SERVER_CONFIG_BASE64_ENV_NAME: str = "FEAST_CONFIG_BASE64"
```

Comment:
nit, can you have this variable name container `FEATURE_STORE_YAML`  so we know what it's referring to? `Config` by itself is a bit ambiguous.